Jenkinsfile

```diff
@@ -96,74 +96,6 @@ pipeline {
                     }
                     post { always { runShell("rm -rf ./*") }}
                 }
-                stage("Try to compile all good integration models") {
-                    when { expression { params.compile_all } }
-                    agent { label 'linux' }
-                    steps {
-                        unstash 'ubuntu-exe'
-                        sh """
-                            git clone --recursive --depth 50 https://github.com/stan-dev/performance-tests-cmdstan
-                        """
-
-                        writeFile(file:"performance-tests-cmdstan/cmdstan/make/local",
-                                  text:"O=0\nCXXFLAGS+=-o/dev/null -S -Wno-unused-command-line-argument")
-                        sh """
-                            cd performance-tests-cmdstan
-                            cd cmdstan; make -j${env.PARALLEL} build; cd ..
-                            cp ../bin/stanc cmdstan/bin/stanc
-                            git clone --depth 1 https://github.com/stan-dev/stanc3
-                            CXX="${CXX}" ./runPerformanceTests.py --runs=0 stanc3/test/integration/good || true
-                        """
-
-                        xunit([GoogleTest(
-                            deleteOutputFiles: false,
-                            failIfNotNew: true,
-                            pattern: 'performance-tests-cmdstan/performance.xml',
-                            skipNoTestFiles: false,
-                            stopProcessingIfError: false)
-                        ])
-                    }
-                    post { always { runShell("rm -rf ./*") }}
-                }
-                stage("Run all models end-to-end") {
-                    agent { label 'linux' }
-                    steps {
-                        unstash 'ubuntu-exe'
-                        sh """
-                            git clone --recursive --depth 50 https://github.com/stan-dev/performance-tests-cmdstan
-                        """
-                        sh """
-                            cd performance-tests-cmdstan
-                            git show HEAD --stat
-                            echo "example-models/regression_tests/mother.stan" > all.tests
-                            cat known_good_perf_all.tests >> all.tests
-                            echo "" >> all.tests
-                            cat shotgun_perf_all.tests >> all.tests
-                            cat all.tests
-                            echo "CXXFLAGS+=-march=core2" > cmdstan/make/local
-                            cd cmdstan; git show HEAD --stat; STANC2=true make -j4 build; cd ..
-                            CXX="${CXX}" ./compare-compilers.sh "--tests-file all.tests --num-samples=10" "\$(readlink -f ../bin/stanc)"
```

Comment:
Is this some sort of compiler benchmark we just aren't doing anymore?

Reply:
Hm, yeah, it previously ran stanc2 tests that all failed since 2.25.

But comparing nightly vs PR compiler is something we should do, just not stanc2. Maybe we can add this as a separate stage.